<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1487</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#widget_2 {
        border: none;                /* Remove all borders */
        border-right: 1px solid black;  /* Right border only */
    }

QPushButton{
	
	color: rgb(0, 0, 0);
	text-align:left;
	height: 50px;
	border:none;
	padding-left: 15px;
	padding-right:15px;
	border-radius:15px;
}

QPushButton:checked{
	
	color: rgb(255, 255, 255);
	background-color: rgb(135, 159, 255);
}


QPushButton:hover:!checked{
	
	
	background-color: rgb(244, 244, 244);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>50</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="dev_btn">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Developer Demographics</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="land_btn">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Languages &amp; Tools</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="career_btn">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string> Career Insights</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="work_btn">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Work Environment</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>439</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="developer">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QWidget" name="chart" native="true">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(223, 213, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="chart2" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="language">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QWidget" name="dev_chart_container" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="fram_top" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="career">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QWidget" name="scat_plt" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="scat_plt2" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="work"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="D:/python/Assistant_app/resources_main.qrc"/>
 </resources>
 <connections/>
</ui>
